.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git
+Dockerfile*
+target
+.cargo
```

Comment:
Let's add `.github` as well, when #4 merges?

Reply:
Also, should we ignore the `README`? It's not big, but it's an extra thing to copy into the docker image, and we don't want to invalidate the cache when it changes?